Keas.Mvc/Controllers/Api/KeySerialsController.cs

```diff
@@ -107,15 +107,17 @@ public async Task<IActionResult> Create([FromBody] CreateKeySerialViewModel mode
             // create key serial
             var keySerial = new KeySerial
             {
-                KeyId = model.KeyId,
-                Number = model.Number,
+                KeyId    = key.Id,
+                TeamId   = key.TeamId,
+                Name     = model.Number,
+                Number   = model.Number,
             };
 
             // add key serial
             key.Serials.Add(keySerial);
             await _context.SaveChangesAsync();
-
-            //await _eventService.TrackCreateKeySerial(key);
+            
+            //await _eventService.TrackCreateKeySerial(keySerial);
```

Comment:
@jcubbage @jpknoll We are not tracking this?

Reply:
No...we are tracking keys and assignments. Not key serials.